website/content/docs/auth/jwt/oidc-providers/secureauth.mdx

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+page_title: OIDC Provider Setup - Auth Methods - SecureAuth
+description: OIDC provider configuration for SecureAuth
+---
+
+## SecureAuth
+
+TODO
```

Comment:
@markafarrell - I'm adding documentation for your contribution here. Feel free to contribute any useful snippets related to configuring the SecureAuth OIDC provider side of things. 

cc: @calvn since I think you configured the SecureAuth instance for our testing :)

Reply:
Ah, I'm also realizing now that SecureAuth group membership support won't land until Vault 1.12. I guess we could still keep this documentation (but remove the group membership bits until they're released) since we've tested the IdP otherwise.

Reply:
Yeah, we could keep it since they both won't land until the 1.12 release.

Reply:
I'm going to put the SecureAuth additions in a separate PR. I'd like to merge and publish the changes that break out the OIDC providers into individual pages more immediately.

Reply:
Removed the SecureAuth docs in 98d5ba9. I'll open a PR with the docs after this is merged.